crates/sui-cost-tables/src/bytecode_tables.rs

```diff
@@ -25,7 +25,7 @@ use move_binary_format::{
 };
 
 /// VM flat fee
-pub const VM_FLAT_FEE: Gas = Gas::new(1_000);
+pub const VM_FLAT_FEE: Gas = Gas::new(2_000);
```

Comment:
as this might need to tuned, maybe its better to put the number in some config or env files?

Reply:
Well this leads to a potential for each validator to start with different values.
Also if we use cfg files here, why not do it for all values in the cost table?

Reply:
I think this constant is a good place to tune this. 

In the future, we might need to change this as the software upgrades, but we will need to sort out that whole problem at a later date.